ansible/roles/jboss_eap_install/tasks/main.yaml

```diff
@@ -1,55 +1,67 @@
-- block:
-  - name: Install packages required for jboss
-    action: "{{ ansible_pkg_mgr }} name={{ item }} state=present"
-    with_items:
-      - unzip
-      - java-1.8.0-openjdk-devel
-      - java-1.8.0-openjdk
-      - java-1.8.0-openjdk-headless
-
-  - name: Ensure base dir that jboss will be installed into and log dir exists
-    file:
-      path: '{{item}}'
-      state: directory
-      owner: '{{jboss_username}}'
-      group: '{{jboss_username}}'
-      mode: 0755
-    with_items:
-        - '{{jboss_base_dir}}'
-        - '{{jboss_log_dir}}'
-
-  - name: Unpack jboss zip into base dir
-    unarchive:
-      copy: True
-      src: '{{jboss_zip_path}}'
-      dest: '{{jboss_base_dir}}/'
-      group: '{{jboss_username}}'
-      mode: "0644"
-      owner: '{{jboss_username}}'
-      list_files: True
-    register: unzip_result
-
-  - name: Set name of jboss install dir based on unzip task
-    set_fact:
-        jboss_install_dir: "{{jboss_base_dir}}/{{unzip_result['files'][0]}}"
+- name: Install packages required for jboss
+  action: "{{ ansible_pkg_mgr }} name={{ item }} state=present"
+  with_items:
+    - unzip
+    - java-1.8.0-openjdk-devel
+    - java-1.8.0-openjdk
+    - java-1.8.0-openjdk-headless
 
-  - debug:
-      msg: install dir is now {{jboss_install_dir}}
+- name: Using chkconfig, must use canonical process name and log dir
+  set_fact:
+      # when we use chkconfig, these vars are determined by the init script
+      jboss_service_name: 'jboss-eap-rhel'
+      jboss_log_dir: '/var/log/jboss-eap'
+  when: not (systemd == True)
```

Comment:
if systemd is a boolean (see previous comment) then this condition can be just `when: not systemd`